src/backoffice/dashboards/examine-management/views/section-view-examine-searchers.ts

```diff
@@ -0,0 +1,306 @@
+				display: flex;
+			}
+			div.flex > uui-button {
+				padding-left: var(--uui-size-space-4);
+				height: 0;
+			}
+
+			uui-input {
+				width: 100%;
+				margin-bottom: var(--uui-size-space-5);
+			}
+
+			uui-table-head-cell {
+				text-transform: capitalize;
+			}
+
+			uui-table-row:last-child uui-table-cell {
+				padding-bottom: 0;
+			}
+
+			uui-table-cell {
+				min-width: 100px;
+			}
+
+			button.bright {
+				font-style: italic;
+				color: var(--uui-color-positive-emphasis);
+			}
+
+			.field-adder {
+				line-height: 0;
+				cursor: pointer;
+				vertical-align: top;
+				background: transparent;
+				border: none;
+			}
+
+			.table-container uui-scroll-container {
+				padding-bottom: var(--uui-size-space-4);
+				max-width: 100%;
+				overflow-x: scroll;
+				flex: 1;
+			}
+
+			.table-container {
+				display: flex;
+				align-items: flex-start;
+			}
+			uui-tag {
+				margin-block: var(--uui-size-5, 15px);
+			}
+
+			.exposed-field uui-button {
+				align-items: center;
+				font-weight: normal;
+				font-size: 10px;
+				height: 10px;
+				width: 10px;
+				margin-top: -5px;
+			}
+
+			.exposed-field-container {
+				display: flex;
+				justify-content: space-between;
+			}
+		`,
+	];
+
+	private _notificationService?: UmbNotificationService;
+	private _modalService?: UmbModalService;
+
+	@property()
+	searcherName!: string;
+
+	@state()
+	private _searchResults?: SearchResult[];
+
+	@state()
+	private _exposedFields?: ExposedSearchResultField[];
+
+	@query('#search-input')
+	private _searchInput!: HTMLInputElement;
+
+	constructor() {
+		super();
+		this.consumeContext('umbNotificationService', (notificationService: UmbNotificationService) => {
+			this._notificationService = notificationService;
+		});
+		this.consumeContext('umbModalService', (modalService: UmbModalService) => {
+			this._modalService = modalService;
+		});
+	}
+
+	private _onNameClick() {
+		const data: UmbNotificationDefaultData = { message: 'TODO: Open editor for this' }; // TODO
+		this._notificationService?.peek('warning', { data });
+	}
+
+	private _onKeyPress(e: KeyboardEvent) {
+		e.key == 'Enter' ? this._onSearch() : undefined;
+	}
+
+	private async _onSearch() {
+		if (!this._searchInput.value.length) return;
+		try {
+			const res = await SearchResource.getSearchSearcherBySearcherNameSearch({
+				searcherName: this.searcherName,
+				query: this._searchInput.value,
+				take: 9999,
+				skip: 0,
+			});
+			this._searchResults = res.items as SearchResult[];
+			this._updateFieldFilter();
+		} catch (e) {
+			if (e instanceof ApiError) {
+				const error = e as ProblemDetails;
+				const data: UmbNotificationDefaultData = { message: error.message ?? 'Could not fetch search results' };
+				this._notificationService?.peek('danger', { data });
+			}
+		}
+	}
+
+	private _updateFieldFilter() {
+		this._searchResults?.map((doc) => {
+			const document = doc.fields?.filter((field) => {
+				return field.name?.toUpperCase() != 'NODENAME';
```

Comment:
Consider using `!==` instead of `!=` for strict compare.

```suggestion
				return field.name?.toUpperCase() !== 'NODENAME';
```